moveit_core/sensor_manager/CMakeLists.txt

```diff
@@ -1 +1 @@
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
+install(DIRECTORY include/ DESTINATION include)
```

Comment:
@mlautman is this best practice? should we ask dirk?

Reply:
Taken from https://index.ros.org/doc/ros2/Contributing/Migration-Guide/.

Reply:
As stated somewhere else, I believe `GNUInstallDirs` is the better approach here - from a maintainer's point of few at least.
@dirk-thomas I believe you were involved with the migration instructions, right?
Did you know about that cmake module or did you deliberately ignore it?
The only possible shortcoming I can see is that you might end up with `lib<arch>` folders instead of `lib` depending on the context

Reply:
Using `GNUInstallDirs` or not is pretty much orthogonal to the migration instructions. You can either use it or not - in ROS 1 as well as in ROS 2 packages.

Reply:
It sounds like switching to ``GNUInstallDirs`` is outside the scope of a first-pass moveit port to ROS2, and @vmayoral 's approach is good